### DISCUSSION TITLE
Move the Project menu option to the first

### DISCUSSION THREAD

**Discussion by nirlob** [godotengine/godot-proposals#10442] [GUI]
By hierarchy I supposed that first menu must be Project before Scene and in it the options 'Open project', 'Close project' and 'Exit'. I don't know much about this project and surely the order is for some reason but my impression is that. I test the changes and show like this. 

![image](https://github.com/user-attachments/assets/a8d1faba-8263-44fc-ac02-1187e2f48936)

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/7700.

Note that reordering the Scene and Project menus was done in https://github.com/godotengine/godot/pull/60083, but it was rejected.
